File: webservice/Frikollection_Api/Frikollection_Api/DTOs/Notification/NotificationDto.cs
namespace Frikollection_Api.DTOs.Notification
{
    public class NotificationDto
    {
        public Guid CollectionId { get; set; } = Guid.NewGuid();
        public string Message { get; set; } = string.Empty;
        public string FollowerNickname { get; set; } = string.Empty;
        public string CollectionName { get; set; } = string.Empty;
        public DateTime CreatedAt { get; set; }
        public bool IsRead { get; set; }
    }
}

File: mobile_desktop_app/FrikollectionMobileDesktop/composeApp/src/commonMain/kotlin/org/example/frikollection_mobile_desktop/models/user/NotificationDto.kt
package org.example.frikollection_mobile_desktop.models.user

import kotlinx.serialization.Serializable

@Serializable
data class NotificationDto(
    val notificationId: String,
    val message: String,
    val followerNickname: String,
    val collectionName: String,
    val createdAt: String,
    val isRead: Boolean
)